modules/apps/forms-and-workflow/dynamic-data-mapping/dynamic-data-mapping-form-builder/src/main/resources/META-INF/resources/js/form_builder_render_rule_condition.js

```diff
@@ -689,6 +689,37 @@ AUI.add(
 				field.render(container);
 
 				instance._conditions[index + '-condition-second-operand-input'] = field;
+			_renderSecondOperandNumeric: function(index, condition, container) {
+				var instance = this;
+
+				var value = '';
+
+				var firstOperand = instance._getFirstOperand(index);
+
+				var secondOperandTypeValue = instance._getSecondOperandTypeValue(index);
+
+				var visible = instance._isNumber(secondOperandTypeValue) && !instance._isFieldList(firstOperand);
+
+				if (condition && instance._isBinaryCondition(index) && visible) {
```

Comment:
Group these expressions with parentheses

Reply:
As we agreed, there is no need to add parentheses if we don't have a comparison.